### DISCUSSION TITLE
Question about reading the data columns from .nda file

### DISCUSSION THREAD

**Discussion by yqdleiyi** [FTHuld/neware_reader#16] [Q&A]
I found that for the result of processing .nda file, the code failed to fetch all columns needed. There should be a column called cycle ID in output dataframe file, but actually not.

**Comment by FTHuld** [comment]
Yes. Unfortunately I haven't managed to find the bytestream matching "cycle ID", but since I've never actually needed it I haven't looked too hard. If you try using the option `testcols == True` it's probably somewhere in the output columns named `'column_1'` or `'column_2'`. However, these columns are just the aggregate bytes left over between the important stuff. They might not contain any useful information, or they might contain the information in a jumbled format (i.e. too many bytes in the column).
